src/components/ContentHighlights/data/constants.js

```diff
@@ -23,18 +30,51 @@ export const MAX_CONTENT_ITEMS_PER_HIGHLIGHT_SET = 12;
 export const HIGHLIGHT_TITLE_MAX_LENGTH = 60;
 // Stepper Step Text that match testing components
 export const STEPPER_STEP_TEXT = {
-  createTitle: 'Create a title for your highlight',
-  selectContent: 'Add content to your highlight',
-  confirmContent: 'Confirm your content selections',
+  HEADER_TEXT: {
+    createTitle: 'Create a title for your highlight',
+    selectContent: 'Add content to your highlight',
+    confirmContent: 'Confirm your content selections',
+  },
+  SUB_TEXT: {
+    createTitle: 'Create a unique title for your highlight. This title is visible to your learners and helps them discovery relevant content.',
+    selectContent: (highlightTitle) => parse(`Select up to <strong>${MAX_CONTENT_ITEMS_PER_HIGHLIGHT_SET}</strong> items for "${highlightTitle}".`),
```

Comment:
Example of using html parser to bold inline text in a single string. Allows us to extract static text elements into its own file even if it renders with formatting element tags or spans.